Formula/magnetix.rb

```diff
@@ -0,0 +1,36 @@
+class Magnetix < Formula
+  desc "Interpreter for Magnetic Scrolls adventures"
+  homepage "http://www.maczentrisch.de/magnetiX/"
+  url "http://www.maczentrisch.de/magnetiX/downloads/magnetiX_src.zip"
+  version "3.1"
+  sha256 "9862c95659c4db0c5cbe604163aefb503e48462c5769692010d8851d7b31c2fb"
+
+  depends_on :macos => :lion
+  depends_on :xcode => :build
```

Comment:
I feel like a MacOS dep makes sense at the top, but I won't fight you too hard over this if you have a strong preference!